extensions/README.md

```diff
@@ -10,7 +10,6 @@ If you write your own extension, just replace `gubbins` with whatever name you c
 The structure of the repo, the content of the `pyproject.toml` files, the `__init__` of the modules... are all skeletons that you must reproduce.
 
 Extensions are controled via:
-* the `DIRACX_EXTENSIONS` environment variable. It should be set to `gubbins,diracx`
 * the entrypoints found in the various `pyproject.toml`
```

Comment:
Did you add somewhere about the new entrypoint ?

Reply:
Done in cdddada